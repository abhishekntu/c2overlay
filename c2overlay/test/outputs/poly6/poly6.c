/****************************************************************/
/*   poly6		                                        */
/****************************************************************/

#ifdef TEST
#include <stdio.h>
#endif

#include<math.h>

#ifdef ALGO
int foo(int t,int x,int z)
{	
int t1_1,t1_2,t1_3,t1_4,t1_5,t1_6,t1_7,t1_8,t1_9,t1_10,t2_1,t2_2,t2_3,t2_4,t2_5,t2_6,t2_7,t2_8,t3_1,t3_2,t3_3,t3_4,t3_5,t3_6,t3_7,t4_1,t4_2,t4_3,t4_4,t5_1,t5_2,t5_3,t6_1,t6_2,t6_3,t7_1,t7_2,t8_1,t8_2,t9_1,t10_1,t11_1;

t1_1 = 32 - z;
t1_2 = z - 72;
t1_3 = 87*z;
t1_4 = -z + 864;
t1_5 = 6*x;
t1_6 = 414*z;
t1_7 = 3456*z;
t1_8 = 8*x + 1728;
t1_9 = 124416*z;
t1_10 = 2985984*z;
t2_1 = x*t1_1;
t2_2 = z*t1_2;
t2_3 = t1_3 + 2592;
t2_4 = z*t1_4;
t2_5 = t1_5 + 4*z;
t2_6 = t1_6 - 20736;
t2_7 = t1_7 - 1492992;
t2_8 = z*t1_8;
t3_1 = t2_1 + t2_2;
t3_2 = z*t2_3;
t3_3 = t2_4 - 186624;
t3_4 = t2_5 - 432;
t3_5 = z*t2_6;
t3_6 = z*t2_7;
t3_7 = x*t2_8;
t4_1 = x*t3_1;
t4_2 = z*t3_3;
t4_3 = z*t3_4;
t4_4 = t3_7 + t1_9;
t5_1 = t4_1 - t3_2;
t5_2 = x*t4_3;
t5_3 = x*t4_4;
t6_1 = x*t5_1;
t6_2 = t5_2 - t3_5;
t6_3 = t5_3 + t1_10;
t7_1 = t6_1 + t4_2;
t7_2 = x*t6_2;
t8_1 = t*t7_1;
t8_2 = t7_2 - t3_6;
t9_1 = t8_1 - t8_2;
t10_1 = t*t9_1;
return(t10_1 - t6_3);

}
#endif


#ifdef TEST
int main(void)
{
	int t,x,z;
 	int result;
  	result = foo(t,x,z);
  	return 0;
}
#endif
