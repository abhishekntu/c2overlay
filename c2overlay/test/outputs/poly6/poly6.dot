digraph graphname {
N29 [color="black", label="mul_Imm_124416_N34_N29"];
N28 [color="black", label="N33N42_mul_N33_add_N42_N28"];
N23 [color="black", label="N18N41_mul_Imm_414_N18_sub_Imm_20736_N41_N23"];
N22 [color="black", label="N15N44_mul_N15_sub_N44_N22"];
N21 [color="black", label="N14N43_add_Imm_216_N14_mul_Imm_8_N43_N21"];
N20 [color="black", label="mul_Imm_6_N17_N20"];
N27 [color="black", label="sub_Imm_864_N31_N27"];
N26 [color="black", label="mul_N39_N26"];
N25 [color="black", label="mul_N38_N25"];
N24 [color="black", label="N19N32_mul_Imm_87_N19_add_Imm_2592_N32_N24"];
N8 [color="black", label="sub_Imm_432_N22_N8"];
N9 [color="black", label="mul_Imm_2985984_N26_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="sub_Imm_32_N40_N4"];
N5 [color="black", label="mul_N45_N5"];
N6 [color="black", label="sub_Imm_72_N46_N6"];
N7 [color="black", label="N29N47_mul_N29_add_N47_N7"];
N12 [color="black", label="N4N8_mul_N4_sub_N8_N12"];
N13 [color="black", label="N5N10_mul_N5_sub_N10_N13"];
N10 [color="black", label="N24N20_mul_Imm_3456_N24_sub_Imm_1492992_N20_N10"];
N11 [color="black", label="N9N30_mul_Imm_4_N9_add_N30_N11"];
N16 [color="black", label="mul_N12_N16"];
N17 [color="black", label="mul_N13_N17"];
N14 [color="black", label="N6N23_mul_N6_sub_Imm_186624_N23_N14"];
N15 [color="black", label="N7N21_mul_N7_sub_N21_N15"];
N18 [color="black", label="mul_N11_N18"];
N19 [color="black", label="N16N25_mul_N16_add_N25_N19"];
N30 [color="black", label="N35N28_mul_N35_sub_N28_N30"];
N31 [color="black", label="N36N27_mul_N36_add_N27_N31"];
N32 [color="black", label="mul_N37_N32"];
N33 [color="black", label="O0_N48_N33"];
N29 -> N19;
N28 -> N12;
N23 -> N18;
N22 -> N28;
N21 -> N16;
N20 -> N11;
N27 -> N14;
N26 -> N30;
N25 -> N28;
N24 -> N17;
N8 -> N26;
N9 -> N31;
N1 -> N12;
N1 -> N15;
N2 -> N30;
N2 -> N31;
N2 -> N19;
N2 -> N28;
N2 -> N22;
N2 -> N13;
N2 -> N20;
N2 -> N21;
N2 -> N7;
N3 -> N27;
N3 -> N29;
N3 -> N6;
N3 -> N9;
N3 -> N16;
N3 -> N23;
N3 -> N10;
N3 -> N4;
N3 -> N5;
N3 -> N25;
N3 -> N11;
N3 -> N17;
N3 -> N32;
N3 -> N18;
N3 -> N26;
N3 -> N24;
N3 -> N14;
N4 -> N7;
N5 -> N13;
N6 -> N32;
N7 -> N22;
N12 -> N15;
N13 -> N12;
N10 -> N5;
N11 -> N8;
N16 -> N19;
N17 -> N22;
N14 -> N25;
N15 -> N33;
N18 -> N30;
N19 -> N31;
N30 -> N13;
N31 -> N15;
N32 -> N7;
}
