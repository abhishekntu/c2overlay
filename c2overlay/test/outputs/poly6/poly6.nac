procedure foo(in s32 t,in s32 x,in s32 z,out s32 D_2449)
{
  localvar s32 D_2447;
  localvar s32 D_2448;
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= ldc 32;
  t1_1 <= sub t1_1,z;
  t1_2 <= sub z,72;
  t1_3 <= mul z,87;
  t1_4 <= ldc 864;
  t1_4 <= sub t1_4,z;
  t1_5 <= mul x,6;
  t1_6 <= mul z,414;
  t1_7 <= mul z,3456;
  D_2447 <= add x,216;
  t1_8 <= mul D_2447,8;
  t1_9 <= mul z,124416;
  t1_10 <= mul z,2985984;
  t2_1 <= mul x,t1_1;
  t2_2 <= mul z,t1_2;
  t2_3 <= add t1_3,2592;
  t2_4 <= mul z,t1_4;
  D_2448 <= mul z,4;
  t2_5 <= add D_2448,t1_5;
  t2_6 <= sub t1_6,20736;
  t2_7 <= sub t1_7,1492992;
  t2_8 <= mul z,t1_8;
  t3_1 <= add t2_1,t2_2;
  t3_2 <= mul z,t2_3;
  t3_3 <= sub t2_4,186624;
  t3_4 <= sub t2_5,432;
  t3_5 <= mul z,t2_6;
  t3_6 <= mul z,t2_7;
  t3_7 <= mul x,t2_8;
  t4_1 <= mul x,t3_1;
  t4_2 <= mul z,t3_3;
  t4_3 <= mul z,t3_4;
  t4_4 <= add t3_7,t1_9;
  t5_1 <= sub t4_1,t3_2;
  t5_2 <= mul x,t4_3;
  t5_3 <= mul x,t4_4;
  t6_1 <= mul x,t5_1;
  t6_2 <= sub t5_2,t3_5;
  t6_3 <= add t5_3,t1_10;
  t7_1 <= add t6_1,t4_2;
  t7_2 <= mul x,t6_2;
  t8_1 <= mul t,t7_1;
  t8_2 <= sub t7_2,t3_6;
  t9_1 <= sub t8_1,t8_2;
  t10_1 <= mul t,t9_1;
  D_2449 <= sub t10_1,t6_3;
}
