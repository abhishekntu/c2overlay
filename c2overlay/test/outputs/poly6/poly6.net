.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.input N3
pinlist: N3
    
.output out:N33
pinlist: N33
    
.clb N29_blk
pinlist: N3 open open open N29 open open open open 
subblock: N29_blk 0 open open open 4 open open open open

.clb N28_blk
pinlist: N2 N22 N25 open N28 open open open open 
subblock: N28_blk 0 1 2 open 4 open open open open
    
.clb N23_blk
pinlist: N3 open open open N23 open open open open 
subblock: N23_blk 0 open open open 4 open open open open

.clb N22_blk
pinlist: N7 N2 N17 open N22 open open open open 
subblock: N22_blk 0 1 2 open 4 open open open open
    
.clb N21_blk
pinlist: N2 open open open N21 open open open open 
subblock: N21_blk 0 open open open 4 open open open open

.clb N20_blk
pinlist: N2 open open open N20 open open open open 
subblock: N20_blk 0 open open open 4 open open open open

.clb N27_blk
pinlist: N3 open open open N27 open open open open 
subblock: N27_blk 0 open open open 4 open open open open

.clb N26_blk
pinlist: N8 N3 open open N26 open open open open 
subblock: N26_blk 0 1 open open 4 open open open open

.clb N25_blk
pinlist: N3 N14 open open N25 open open open open 
subblock: N25_blk 0 1 open open 4 open open open open

.clb N24_blk
pinlist: N3 open open open N24 open open open open 
subblock: N24_blk 0 open open open 4 open open open open

.clb N8_blk
pinlist: N11 open open open N8 open open open open 
subblock: N8_blk 0 open open open 4 open open open open

.clb N9_blk
pinlist: N3 open open open N9 open open open open 
subblock: N9_blk 0 open open open 4 open open open open

.clb N4_blk
pinlist: N3 open open open N4 open open open open 
subblock: N4_blk 0 open open open 4 open open open open

.clb N5_blk
pinlist: N10 N3 open open N5 open open open open 
subblock: N5_blk 0 1 open open 4 open open open open

.clb N6_blk
pinlist: N3 open open open N6 open open open open 
subblock: N6_blk 0 open open open 4 open open open open

.clb N7_blk
pinlist: N4 N2 N32 open N7 open open open open 
subblock: N7_blk 0 1 2 open 4 open open open open
    
.clb N12_blk
pinlist: N1 N13 N28 open N12 open open open open 
subblock: N12_blk 0 1 2 open 4 open open open open
    
.clb N13_blk
pinlist: N5 N2 N30 open N13 open open open open 
subblock: N13_blk 0 1 2 open 4 open open open open
    
.clb N10_blk
pinlist: N3 open open open N10 open open open open 
subblock: N10_blk 0 open open open 4 open open open open

.clb N11_blk
pinlist: N3 N20 open open N11 open open open open 
subblock: N11_blk 0 1 open open 4 open open open open

.clb N16_blk
pinlist: N3 N21 open open N16 open open open open 
subblock: N16_blk 0 1 open open 4 open open open open

.clb N17_blk
pinlist: N3 N24 open open N17 open open open open 
subblock: N17_blk 0 1 open open 4 open open open open

.clb N14_blk
pinlist: N3 N27 open open N14 open open open open 
subblock: N14_blk 0 1 open open 4 open open open open

.clb N15_blk
pinlist: N1 N12 N31 open N33 open open open open 
subblock: N15_blk 0 1 2 open 4 open open open open
    
.clb N18_blk
pinlist: N3 N23 open open N18 open open open open 
subblock: N18_blk 0 1 open open 4 open open open open

.clb N19_blk
pinlist: N2 N16 N29 open N19 open open open open 
subblock: N19_blk 0 1 2 open 4 open open open open
    
.clb N30_blk
pinlist: N2 N18 N26 open N30 open open open open 
subblock: N30_blk 0 1 2 open 4 open open open open
    
.clb N31_blk
pinlist: N9 N2 N19 open N31 open open open open 
subblock: N31_blk 0 1 2 open 4 open open open open
    
.clb N32_blk
pinlist: N6 N3 open open N32 open open open open 
subblock: N32_blk 0 1 open open 4 open open open open

