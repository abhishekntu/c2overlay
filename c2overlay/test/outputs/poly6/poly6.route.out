Array size: 6 x 6 logic blocks.

Routing:

Net 0 (N1)

SOURCE (0,3)  Pad: 1  
  OPIN (0,3)  Pad: 1  
 CHANY (0,3)  Track: 3  
 CHANX (1,2)  Track: 2  
 CHANX (2,2)  Track: 2  
 CHANX (3,2)  Track: 2  
  IPIN (3,3)  Pin: 2  
  SINK (3,3)  Class: 0  
 CHANX (3,2)  Track: 2  
  IPIN (3,2)  Pin: 0  
  SINK (3,2)  Class: 0  


Net 1 (N2)

SOURCE (2,0)  Pad: 1  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 3  
  IPIN (2,1)  Pin: 2  
  SINK (2,1)  Class: 0  
 CHANX (2,0)  Track: 3  
 CHANY (1,1)  Track: 2  
 CHANY (1,2)  Track: 2  
  IPIN (2,2)  Pin: 3  
  SINK (2,2)  Class: 0  
 CHANY (1,2)  Track: 2  
 CHANY (1,3)  Track: 2  
 CHANY (1,4)  Track: 2  
  IPIN (2,4)  Pin: 3  
  SINK (2,4)  Class: 0  
 CHANY (1,3)  Track: 2  
 CHANX (2,3)  Track: 2  
 CHANX (3,3)  Track: 2  
  IPIN (3,4)  Pin: 2  
  SINK (3,4)  Class: 0  
 CHANX (3,3)  Track: 2  
 CHANX (4,3)  Track: 2  
  IPIN (4,4)  Pin: 2  
  SINK (4,4)  Class: 0  
 CHANX (4,3)  Track: 2  
 CHANX (5,3)  Track: 2  
  IPIN (5,4)  Pin: 2  
  SINK (5,4)  Class: 0  
 CHANX (4,3)  Track: 2  
  IPIN (4,3)  Pin: 0  
  SINK (4,3)  Class: 0  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 2  
 CHANX (3,0)  Track: 2  
 CHANX (4,0)  Track: 2  
  IPIN (4,1)  Pin: 2  
  SINK (4,1)  Class: 0  
 CHANX (4,0)  Track: 2  
 CHANX (5,0)  Track: 2  
  IPIN (5,1)  Pin: 2  
  SINK (5,1)  Class: 0  


Net 2 (N3)

SOURCE (5,0)  Pad: 1  
  OPIN (5,0)  Pad: 1  
 CHANX (5,0)  Track: 3  
 CHANY (4,1)  Track: 2  
 CHANY (4,2)  Track: 2  
  IPIN (5,2)  Pin: 3  
  SINK (5,2)  Class: 0  
 CHANY (4,2)  Track: 2  
 CHANY (4,3)  Track: 2  
  IPIN (5,3)  Pin: 3  
  SINK (5,3)  Class: 0  
 CHANY (4,2)  Track: 2  
 CHANX (5,2)  Track: 2  
 CHANX (6,2)  Track: 2  
  IPIN (6,3)  Pin: 2  
  SINK (6,3)  Class: 0  
 CHANX (5,2)  Track: 2  
 CHANY (5,3)  Track: 2  
 CHANY (5,4)  Track: 2  
  IPIN (6,4)  Pin: 3  
  SINK (6,4)  Class: 0  
 CHANY (4,2)  Track: 2  
  IPIN (4,2)  Pin: 1  
  SINK (4,2)  Class: 0  
 CHANY (5,4)  Track: 2  
 CHANY (5,5)  Track: 2  
  IPIN (6,5)  Pin: 3  
  SINK (6,5)  Class: 0  
 CHANY (5,5)  Track: 2  
  IPIN (5,5)  Pin: 1  
  SINK (5,5)  Class: 0  
 CHANY (5,5)  Track: 2  
 CHANX (5,5)  Track: 3  
 CHANX (4,5)  Track: 3  
  IPIN (4,5)  Pin: 0  
  SINK (4,5)  Class: 0  
 CHANX (4,5)  Track: 3  
 CHANX (3,5)  Track: 3  
  IPIN (3,5)  Pin: 0  
  SINK (3,5)  Class: 0  
 CHANX (3,5)  Track: 3  
 CHANX (2,5)  Track: 3  
  IPIN (2,5)  Pin: 0  
  SINK (2,5)  Class: 0  
 CHANY (4,2)  Track: 2  
 CHANX (4,2)  Track: 3  
 CHANX (3,2)  Track: 3  
 CHANX (2,2)  Track: 3  
  IPIN (2,3)  Pin: 2  
  SINK (2,3)  Class: 0  
 CHANX (2,2)  Track: 3  
 CHANX (1,2)  Track: 3  
  IPIN (1,3)  Pin: 2  
  SINK (1,3)  Class: 0  
 CHANX (2,5)  Track: 3  
 CHANY (1,5)  Track: 3  
 CHANY (1,4)  Track: 3  
  IPIN (1,4)  Pin: 1  
  SINK (1,4)  Class: 0  
 CHANX (1,2)  Track: 3  
  IPIN (1,2)  Pin: 0  
  SINK (1,2)  Class: 0  
 CHANX (1,2)  Track: 3  
 CHANY (0,2)  Track: 3  
 CHANY (0,1)  Track: 3  
  IPIN (1,1)  Pin: 3  
  SINK (1,1)  Class: 0  
 CHANY (4,1)  Track: 2  
 CHANX (4,1)  Track: 3  
 CHANX (3,1)  Track: 3  
  IPIN (3,1)  Pin: 0  
  SINK (3,1)  Class: 0  
  OPIN (5,0)  Pad: 1  
 CHANX (5,0)  Track: 0  
 CHANX (6,0)  Track: 0  
  IPIN (6,1)  Pin: 2  
  SINK (6,1)  Class: 0  


Net 3 (N29)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 5  
 CHANY (1,1)  Track: 1  
  IPIN (2,1)  Pin: 3  
  SINK (2,1)  Class: 0  


Net 4 (N28)

SOURCE (4,3)  Class: 1  
  OPIN (4,3)  Pin: 7  
 CHANY (3,3)  Track: 3  
  IPIN (3,3)  Pin: 1  
  SINK (3,3)  Class: 0  


Net 5 (N23)

SOURCE (6,5)  Class: 1  
  OPIN (6,5)  Pin: 6  
 CHANX (6,4)  Track: 3  
  IPIN (6,4)  Pin: 0  
  SINK (6,4)  Class: 0  


Net 6 (N22)

SOURCE (4,4)  Class: 1  
  OPIN (4,4)  Pin: 6  
 CHANX (4,3)  Track: 0  
 CHANY (4,3)  Track: 1  
  IPIN (4,3)  Pin: 1  
  SINK (4,3)  Class: 0  


Net 7 (N21)

SOURCE (4,1)  Class: 1  
  OPIN (4,1)  Pin: 7  
 CHANY (3,1)  Track: 3  
  IPIN (3,1)  Pin: 1  
  SINK (3,1)  Class: 0  


Net 8 (N20)

SOURCE (5,1)  Class: 1  
  OPIN (5,1)  Pin: 5  
 CHANY (5,1)  Track: 2  
  IPIN (6,1)  Pin: 3  
  SINK (6,1)  Class: 0  


Net 9 (N27)

SOURCE (4,2)  Class: 1  
  OPIN (4,2)  Pin: 5  
 CHANY (4,2)  Track: 1  
 CHANX (5,1)  Track: 0  
  IPIN (5,2)  Pin: 2  
  SINK (5,2)  Class: 0  


Net 10 (N26)

SOURCE (6,3)  Class: 1  
  OPIN (6,3)  Pin: 7  
 CHANY (5,3)  Track: 0  
 CHANY (5,4)  Track: 0  
  IPIN (5,4)  Pin: 1  
  SINK (5,4)  Class: 0  


Net 11 (N25)

SOURCE (5,3)  Class: 1  
  OPIN (5,3)  Pin: 6  
 CHANX (5,2)  Track: 1  
 CHANX (4,2)  Track: 1  
  IPIN (4,3)  Pin: 2  
  SINK (4,3)  Class: 0  


Net 12 (N24)

SOURCE (5,5)  Class: 1  
  OPIN (5,5)  Pin: 7  
 CHANY (4,5)  Track: 3  
  IPIN (4,5)  Pin: 1  
  SINK (4,5)  Class: 0  


Net 13 (N8)

SOURCE (6,2)  Class: 1  
  OPIN (6,2)  Pin: 4  
 CHANX (6,2)  Track: 0  
 CHANY (6,3)  Track: 0  
  IPIN (6,3)  Pin: 1  
  SINK (6,3)  Class: 0  


Net 14 (N9)

SOURCE (1,2)  Class: 1  
  OPIN (1,2)  Pin: 5  
 CHANY (1,2)  Track: 1  
 CHANX (2,1)  Track: 0  
  IPIN (2,2)  Pin: 2  
  SINK (2,2)  Class: 0  


Net 15 (N4)

SOURCE (1,4)  Class: 1  
  OPIN (1,4)  Pin: 5  
 CHANY (1,4)  Track: 1  
 CHANX (2,3)  Track: 0  
  IPIN (2,4)  Pin: 2  
  SINK (2,4)  Class: 0  


Net 16 (N5)

SOURCE (3,5)  Class: 1  
  OPIN (3,5)  Pin: 6  
 CHANX (3,4)  Track: 3  
  IPIN (3,4)  Pin: 0  
  SINK (3,4)  Class: 0  


Net 17 (N6)

SOURCE (1,3)  Class: 1  
  OPIN (1,3)  Pin: 5  
 CHANY (1,3)  Track: 1  
  IPIN (2,3)  Pin: 3  
  SINK (2,3)  Class: 0  


Net 18 (N7)

SOURCE (2,4)  Class: 1  
  OPIN (2,4)  Pin: 4  
 CHANX (2,4)  Track: 0  
 CHANX (3,4)  Track: 0  
 CHANX (4,4)  Track: 0  
  IPIN (4,4)  Pin: 0  
  SINK (4,4)  Class: 0  


Net 19 (N12)

SOURCE (3,3)  Class: 1  
  OPIN (3,3)  Pin: 6  
 CHANX (3,2)  Track: 1  
 CHANY (2,2)  Track: 1  
  IPIN (3,2)  Pin: 3  
  SINK (3,2)  Class: 0  


Net 20 (N13)

SOURCE (3,4)  Class: 1  
  OPIN (3,4)  Pin: 6  
 CHANX (3,3)  Track: 3  
  IPIN (3,3)  Pin: 0  
  SINK (3,3)  Class: 0  


Net 21 (N10)

SOURCE (2,5)  Class: 1  
  OPIN (2,5)  Pin: 5  
 CHANY (2,5)  Track: 2  
  IPIN (3,5)  Pin: 3  
  SINK (3,5)  Class: 0  


Net 22 (N11)

SOURCE (6,1)  Class: 1  
  OPIN (6,1)  Pin: 4  
 CHANX (6,1)  Track: 3  
  IPIN (6,2)  Pin: 2  
  SINK (6,2)  Class: 0  


Net 23 (N16)

SOURCE (3,1)  Class: 1  
  OPIN (3,1)  Pin: 7  
 CHANY (2,1)  Track: 3  
  IPIN (2,1)  Pin: 1  
  SINK (2,1)  Class: 0  


Net 24 (N17)

SOURCE (4,5)  Class: 1  
  OPIN (4,5)  Pin: 6  
 CHANX (4,4)  Track: 3  
 CHANY (3,4)  Track: 3  
  IPIN (4,4)  Pin: 3  
  SINK (4,4)  Class: 0  


Net 25 (N14)

SOURCE (5,2)  Class: 1  
  OPIN (5,2)  Pin: 4  
 CHANX (5,2)  Track: 3  
  IPIN (5,3)  Pin: 2  
  SINK (5,3)  Class: 0  


Net 26 (N33)

SOURCE (3,2)  Class: 1  
  OPIN (3,2)  Pin: 5  
 CHANY (3,2)  Track: 1  
 CHANY (3,1)  Track: 1  
 CHANX (3,0)  Track: 1  
  IPIN (3,0)  Pad: 0  
  SINK (3,0)  Pad: 0  


Net 27 (N18)

SOURCE (6,4)  Class: 1  
  OPIN (6,4)  Pin: 4  
 CHANX (6,4)  Track: 1  
 CHANX (5,4)  Track: 1  
  IPIN (5,4)  Pin: 0  
  SINK (5,4)  Class: 0  


Net 28 (N19)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 4  
 CHANX (2,1)  Track: 2  
 CHANY (2,2)  Track: 2  
  IPIN (2,2)  Pin: 1  
  SINK (2,2)  Class: 0  


Net 29 (N30)

SOURCE (5,4)  Class: 1  
  OPIN (5,4)  Pin: 7  
 CHANY (4,4)  Track: 3  
 CHANX (4,3)  Track: 3  
 CHANY (3,4)  Track: 2  
  IPIN (3,4)  Pin: 1  
  SINK (3,4)  Class: 0  


Net 30 (N31)

SOURCE (2,2)  Class: 1  
  OPIN (2,2)  Pin: 5  
 CHANY (2,2)  Track: 3  
 CHANX (3,1)  Track: 2  
  IPIN (3,2)  Pin: 2  
  SINK (3,2)  Class: 0  


Net 31 (N32)

SOURCE (2,3)  Class: 1  
  OPIN (2,3)  Pin: 5  
 CHANY (2,3)  Track: 2  
 CHANY (2,4)  Track: 2  
  IPIN (2,4)  Pin: 1  
  SINK (2,4)  Class: 0  
